After signup/after-signup.html

```diff
@@ -4,11 +4,109 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!--FONT AWESOME LIBRARY STYLE LINK-->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" />
+    <!--STYLE CSS FILE-->
     <link rel="stylesheet" href="after-signup.css" />
     <title>After Signup</title>
 </head>
 <body>
-    
+    <!-- HEADER START -->
+    <header>
+        <nav>
+            <i class="fas fa-bars"></i>
+        </nav>
+        <div class="img-profile">
+            <i class="far fa-bell"></i>
+            <img src="../images/Ellipse 36.png">
+        </div>
+    </header>
+    <!--HEADER END-->
+    <main>
+        <!--HEADING SECTION START-->
+        <section id="heading">
+            <h1>Hello, Lisa <img src="../images/hand.svg"></h1>
+            <p>What do you wanna donate today?</p>
+        </section>
+        <!--HEADING SECTION END-->
+        <!--SEARCH SECTION START-->
+        <section id="search">
+            <form>
+                <i class="fas fa-search"></i>
+                <input type="text" placeholder="Search here">
+                <i class="fas fa-microphone-alt"></i>
+                <label></label>
+                <i class="fas fa-sliders-h"></i>
+            </form>
+        </section>
+        <!--SEARCH SECTION END-->
+        <!--CAMPAIGN SECTION START-->
+        <section id="campaign">
+            <h2>Do you really have a creative <label>idea?</label></h2>
+            <a href="see-blog.html">start campaign</a>
+        </section>
+        <!--CAMPAIGN SECTION END-->
+        <!--FRAME SECTION START-->
+        <section id="frame">
+            <p>categories</p>
+            <div class="boxes">
+                <div class="box">
+                    <i class="fas fa-th-large purple"></i>
+                    <span>all</span>
+                </div>
+                <div class="box">
+                    <i class="fas fa-volume-up green"></i>
+                    <span>campaigns</span>
+                </div>
+                <div class="box">
+                    <i class="fas fa-toolbox blue"></i>
+                    <span>donate goods</span>
+                </div>
+                <div class="box">
+                    <i class="fas fa-donate orange"></i>
+                    <span>charity</span>
+                </div>
+            </div>
+        </section>
+        <!--FRAME SECTION END-->
+        <!--GROUP SECTION START-->
+        <section id="group">
+            <p>my campaign</p>
+            <figure>
+                <img src="../images/unsplash_yg1Zayn0Few.png"
+            </figure>
```

Comment:
Add 'save' icon for a single campaign

Reply:
![image](https://user-images.githubusercontent.com/77115941/152850716-09597822-03fb-4c2e-bee3-efd748d780bb.png)